pandas/core/arrays/base.py

```diff
@@ -35,6 +38,15 @@ class ExtensionArray(object):
     * _can_hold_na
     * _formatting_values
 
+    Some methods require casting the ExtensionArray to an ndarray of Python
+    objects, which may be expensive. When performance is a concern, we highly
+    recommend overriding the following methods.
```

Comment:
Maybe clarify that those have a default working implementation, but do a `self.astype(object)` ?